app/services/foreman/access_permissions.rb

```diff
@@ -394,18 +394,21 @@
   end
 
   permission_set.security_block :host_editing do |map|
-    map.permission :edit_classes,   {:host_editing => [:edit_classes],
+    map.permission :edit_classes, {:host_editing => [:edit_classes],
                                      :"api/v2/host_classes" => [:index, :create, :destroy]
```

Comment:
Nitpick, the symbols on the second lines aren't aligned to the first (`:host_editing` vs `:"api...`)